icons/processwire/processwire-plain-wordmark.svg

```diff
@@ -0,0 +1,5 @@
+  </path>
```

Comment:
Change path to be self-closing.
Use hexcode for fill instead of rgb().
```suggestion
  <path fill="#e81d60" d="M 23.87 57.088 C 24.72 58.354 25.455 59.924 25.732 61.353 C 26.081 62.894 26.093 64.5 25.832 66.063 C 25.544 67.626 24.988 69.146 24.129 70.5 C 23.459 71.641 22.245 72.981 20.85 73.965 C 19.547 74.891 18.066 75.558 16.523 75.924 C 15.752 76.107 14.966 76.216 14.184 76.245 C 13.002 76.291 11.545 76.216 10.472 75.738 C 10.002 75.54 9.525 75.218 9.15 74.819 C 8.883 74.558 8.493 73.99 8.307 73.438 C 8.126 72.936 8.068 72.471 8.046 72.058 C 8.005 71.435 8.046 71.227 8.046 70.017 L 8.046 66.236 C 8.046 65.663 8.015 65.05 8.031 64.39 C 8.043 63.499 8.114 62.926 8.381 62.158 C 8.647 61.396 9.087 60.722 9.624 60.163 C 10.406 59.355 11.157 58.982 12.035 58.681 L 12.051 58.675 C 13.403 58.33 14.452 58.349 15.772 58.867 C 17.077 59.393 18.067 60.354 18.641 61.506 C 19.163 62.549 19.361 63.782 19.209 64.927 C 19.056 65.959 18.607 66.881 17.82 67.629 C 16.897 68.49 15.808 68.926 14.704 69.003 C 13.983 69.058 13.376 68.97 12.69 68.758 C 12.476 68.672 12.107 68.43 12.272 67.765 L 12.44 66.892 C 12.455 66.814 12.525 66.765 12.602 66.769 C 13.102 66.803 13.659 66.753 14.165 66.584 C 14.403 66.5 14.756 66.357 15.148 66.063 C 15.464 65.827 16.29 64.896 16.13 63.647 C 16.103 63.434 15.986 62.636 15.286 62.051 C 14.633 61.509 13.891 61.498 13.675 61.498 C 12.966 61.498 12.32 61.794 11.86 62.283 C 11.223 62.946 11.147 63.559 11.116 64.451 C 11.11 64.995 11.132 65.593 11.132 66.253 C 11.132 68.589 11.134 67.388 11.123 71.888 C 11.156 72.379 11.272 72.563 11.272 72.563 C 11.441 72.818 11.595 73.007 12.331 73.124 C 12.826 73.207 13.532 73.224 14.064 73.207 C 14.662 73.188 15.247 73.101 15.829 72.962 C 16.98 72.686 18.1 72.188 19.081 71.498 C 20.171 70.72 21.091 69.683 21.536 68.898 C 22.18 67.887 22.595 66.736 22.825 65.554 C 23.011 64.373 23.011 63.16 22.749 62.017 C 22.565 61.011 21.935 59.639 21.354 58.824 C 20.784 57.972 19.956 57.109 18.929 56.411 C 17.518 55.455 16.197 55.11 15.891 55.034 C 14.449 54.678 13.253 54.752 12.623 54.827 C 10.829 55.035 8.941 55.841 7.47 57.145 C 6.103 58.356 5.228 59.814 4.813 61.211 C 4.599 61.918 4.477 62.591 4.431 63.222 C 4.339 64.386 4.4 65.645 4.537 66.397 C 4.599 66.776 4.659 67.268 4.829 67.921 C 4.845 67.97 4.829 68.028 4.798 68.07 C 4.19 68.752 3.399 68.984 2.917 68.721 C 2.82 68.666 2.633 68.537 2.498 68.217 C 2.498 68.217 1.838 66.551 1.64 65.226 C 1.45 63.92 1.38 62.186 1.909 60.315 C 2.432 58.442 3.618 56.478 5.416 54.874 C 7.356 53.141 9.717 52.102 12.474 51.806 C 15.117 51.532 18.106 52.159 20.686 53.926 C 21.928 54.747 23.165 56.016 23.908 57.126 L 23.87 57.088 Z M 106.955 56.868 C 107.108 56.966 107.219 57.146 107.244 57.338 C 107.269 57.533 107.187 57.87 107.056 58.103 C 106.805 58.552 106.253 58.967 105.727 59.101 C 105.533 59.152 105.277 59.164 105.128 59.131 C 104.904 59.079 104.76 58.973 104.669 58.788 C 104.613 58.673 104.609 58.65 104.61 58.485 C 104.612 58.072 104.817 57.655 105.187 57.309 C 105.57 56.953 106.012 56.763 106.468 56.758 C 106.699 56.756 106.822 56.783 106.955 56.868 Z M 114.3 60.19 C 114.716 60.19 115.031 60.311 115.285 60.566 C 115.559 60.841 115.677 61.175 115.632 61.549 C 115.576 62.027 115.394 62.408 115.045 62.774 C 114.757 63.078 114.477 63.289 114.119 63.474 C 113.916 63.577 113.426 63.75 113.406 63.726 C 113.398 63.717 113.353 63.616 113.306 63.501 L 113.221 63.293 L 113.38 63.195 C 113.599 63.061 113.78 62.88 113.855 62.72 C 114.009 62.39 113.902 62.056 113.596 61.919 C 113.235 61.756 112.567 61.831 112.03 62.094 C 111.515 62.346 110.928 62.95 110.577 63.589 C 110.54 63.655 110.062 64.669 109.516 65.841 C 108.968 67.014 108.496 68.018 108.464 68.075 C 108.346 68.287 108.261 68.391 108.174 68.433 C 108.095 68.471 108.081 68.472 107.949 68.451 C 107.523 68.386 107.17 68.03 107.047 67.545 C 106.973 67.25 106.985 66.87 107.082 66.398 L 107.118 66.226 L 106.854 66.473 C 106.269 67.019 105.889 67.312 105.318 67.655 C 104.697 68.029 104.08 68.274 103.486 68.383 C 103.033 68.467 102.697 68.483 102.41 68.438 C 101.821 68.347 101.378 67.988 101.195 67.454 C 101.058 67.057 101.078 66.603 101.253 66.088 C 101.287 65.993 101.669 65.13 102.106 64.169 C 102.566 63.156 102.919 62.354 102.947 62.262 C 102.972 62.174 102.997 62.035 103.002 61.952 C 103.011 61.821 103.007 61.793 102.969 61.72 C 102.913 61.611 102.857 61.573 102.719 61.548 C 102.517 61.511 102.207 61.563 101.901 61.687 L 101.739 61.753 L 101.619 61.544 C 101.501 61.339 101.5 61.335 101.538 61.308 C 101.708 61.181 102.894 60.569 103.196 60.451 C 103.418 60.365 103.738 60.277 104 60.23 C 104.269 60.182 104.749 60.185 104.94 60.237 C 105.478 60.385 105.726 60.718 105.695 61.253 C 105.682 61.488 105.625 61.758 105.535 62.019 C 105.495 62.137 105.131 62.952 104.729 63.831 C 104.326 64.711 103.968 65.493 103.934 65.57 C 103.844 65.772 103.776 66.024 103.763 66.199 C 103.753 66.332 103.757 66.361 103.793 66.437 C 103.845 66.544 103.93 66.604 104.103 66.66 C 104.212 66.694 104.276 66.701 104.487 66.7 C 104.906 66.699 105.253 66.603 105.731 66.356 C 106.258 66.084 106.713 65.746 107.343 65.157 L 107.498 65.013 L 107.607 64.734 C 107.784 64.286 108.147 63.466 108.496 62.729 C 108.69 62.318 108.749 62.132 108.749 61.923 C 108.749 61.75 108.696 61.65 108.572 61.588 C 108.488 61.546 108.456 61.541 108.282 61.544 C 108.062 61.547 107.889 61.586 107.634 61.687 L 107.477 61.751 L 107.358 61.543 C 107.242 61.339 107.24 61.335 107.278 61.308 C 107.421 61.201 108.477 60.647 108.79 60.516 C 109.459 60.23 110.156 60.126 110.647 60.234 C 110.845 60.279 111.097 60.406 111.202 60.515 C 111.411 60.732 111.481 61.048 111.415 61.475 C 111.393 61.617 111.394 61.694 111.417 61.63 C 111.432 61.587 111.894 61.176 112.105 61.017 C 112.776 60.515 113.354 60.273 114.059 60.203 C 114.143 60.195 114.223 60.19 114.3 60.19 Z M 97.919 60.191 C 98.145 60.19 98.445 60.223 98.676 60.282 C 99.24 60.426 99.717 60.723 99.963 61.081 C 100.248 61.495 100.394 62.191 100.335 62.87 C 100.223 64.17 99.654 65.39 98.726 66.32 C 97.782 67.268 96.236 68.097 94.932 68.355 C 94.518 68.438 94.301 68.46 93.903 68.459 C 93.398 68.457 93.061 68.393 92.685 68.227 C 92.341 68.077 92.029 67.81 91.811 67.478 C 91.746 67.381 91.687 67.303 91.678 67.306 C 91.669 67.308 91.578 67.364 91.475 67.431 C 91.036 67.716 90.334 68.086 89.986 68.217 C 89.751 68.305 89.449 68.382 89.182 68.422 C 88.937 68.458 88.421 68.474 88.206 68.451 C 87.75 68.402 87.27 68.261 86.962 68.084 C 86.669 67.915 86.371 67.627 86.218 67.365 C 86 66.994 85.902 66.501 85.928 65.918 C 85.958 65.22 86.075 64.728 86.338 64.179 C 86.524 63.792 86.725 63.481 87.313 62.674 C 87.629 62.24 87.657 62.187 87.671 62.019 C 87.68 61.912 87.675 61.879 87.64 61.807 C 87.56 61.641 87.355 61.558 87.069 61.576 C 86.95 61.583 86.876 61.602 86.709 61.669 L 86.498 61.752 L 86.389 61.545 C 86.28 61.341 86.279 61.337 86.317 61.309 C 86.492 61.181 87.645 60.586 87.948 60.468 C 88.592 60.217 89.237 60.133 89.705 60.237 C 89.877 60.275 90.12 60.389 90.218 60.477 C 90.477 60.712 90.538 61.122 90.403 61.711 C 90.297 62.174 90.174 62.409 89.65 63.158 C 88.816 64.349 88.737 64.476 88.603 64.862 C 88.456 65.283 88.387 65.853 88.451 66.122 C 88.546 66.525 88.828 66.711 89.344 66.712 C 89.754 66.712 90.162 66.581 90.481 66.347 C 90.886 66.048 91.334 65.417 91.79 64.506 C 91.946 64.195 92.477 63.047 93.211 61.433 C 93.75 60.25 93.804 60.173 94.077 60.197 C 94.575 60.241 95.064 60.629 95.23 61.11 C 95.328 61.394 95.32 61.684 95.202 62.141 C 95.096 62.554 95.068 62.621 94.458 63.908 C 94.228 64.396 94.008 64.876 93.971 64.976 C 93.82 65.38 93.747 65.709 93.745 66.002 C 93.745 66.179 93.748 66.208 93.794 66.301 C 93.961 66.641 94.336 66.768 94.921 66.684 C 95.514 66.601 95.995 66.352 96.459 65.892 C 97.183 65.176 97.66 64.185 97.831 63.039 C 97.864 62.82 97.864 62.093 97.831 61.833 C 97.776 61.385 97.617 60.656 97.546 60.524 C 97.527 60.488 97.511 60.439 97.511 60.415 C 97.511 60.347 97.563 60.266 97.628 60.23 C 97.678 60.204 97.786 60.191 97.919 60.191 Z M 121.444 60.201 C 121.93 60.199 122.262 60.265 122.575 60.424 C 122.95 60.614 123.229 60.969 123.341 61.398 C 123.497 61.99 123.297 62.808 122.849 63.423 C 122.632 63.721 122.192 64.16 121.832 64.438 C 120.814 65.225 119.102 65.948 117.99 66.062 L 117.789 66.084 L 117.825 66.142 C 118 66.418 118.189 66.595 118.453 66.724 C 119.272 67.127 120.792 67.07 122.076 66.591 C 123.412 66.091 124.751 65.13 125.938 63.82 C 126.025 63.726 126.102 63.647 126.112 63.647 C 126.122 63.647 126.212 63.715 126.315 63.798 L 126.5 63.947 L 126.359 64.167 C 125.569 65.388 124.577 66.388 123.454 67.093 C 122.246 67.851 120.926 68.298 119.49 68.434 C 119.201 68.461 118.329 68.473 118.102 68.452 C 117.116 68.363 116.447 68.166 115.905 67.806 C 115.299 67.401 115.01 66.889 114.956 66.121 C 114.931 65.773 115.01 65.198 115.144 64.758 C 115.666 63.036 117.264 61.415 119.194 60.649 C 119.93 60.357 120.697 60.204 121.444 60.201 Z M 120.471 61.845 C 120.115 61.848 119.813 61.943 119.482 62.159 C 119.305 62.274 118.946 62.611 118.774 62.825 C 118.369 63.326 117.955 64.173 117.799 64.814 L 117.787 64.868 L 117.958 64.855 C 118.052 64.849 118.203 64.83 118.291 64.814 C 118.633 64.756 119.01 64.627 119.311 64.463 C 120.182 63.987 120.879 63.211 121.031 62.548 C 121.089 62.287 121.059 62.107 120.934 61.979 C 120.828 61.871 120.735 61.843 120.471 61.845 Z M 53.62 61.242 C 54.405 61.981 54.81 62.998 54.834 64.286 C 54.81 65.575 54.405 66.591 53.62 67.331 C 52.835 68.071 51.809 68.445 50.545 68.46 C 49.28 68.445 48.254 68.071 47.469 67.331 C 46.683 66.591 46.278 65.575 46.255 64.286 C 46.278 62.998 46.683 61.981 47.469 61.242 C 48.254 60.502 49.28 60.128 50.545 60.113 C 51.809 60.128 52.835 60.502 53.62 61.242 Z M 61.763 60.37 C 62.253 60.549 62.675 60.85 63.026 61.277 L 61.679 62.38 C 61.505 62.171 61.29 62.007 61.039 61.895 C 60.787 61.783 60.498 61.727 60.172 61.726 C 59.496 61.74 58.95 61.982 58.528 62.45 C 58.105 62.919 57.887 63.531 57.874 64.286 C 57.888 65.041 58.102 65.653 58.519 66.122 C 58.936 66.591 59.47 66.833 60.118 66.847 C 60.474 66.845 60.787 66.768 61.066 66.62 C 61.343 66.472 61.573 66.265 61.754 65.993 L 63.217 67.082 C 62.875 67.552 62.447 67.901 61.937 68.127 C 61.427 68.352 60.896 68.462 60.341 68.46 C 59.077 68.445 58.051 68.071 57.265 67.331 C 56.48 66.591 56.075 65.575 56.052 64.286 C 56.075 62.998 56.48 61.981 57.265 61.242 C 58.051 60.502 59.077 60.128 60.341 60.113 C 60.798 60.107 61.274 60.192 61.763 60.37 Z M 76.169 60.304 C 76.585 60.434 76.962 60.644 77.303 60.939 L 76.098 62.255 C 75.95 62.076 75.757 61.941 75.52 61.855 C 75.283 61.768 75.053 61.725 74.831 61.726 C 74.566 61.724 74.309 61.785 74.058 61.913 C 73.807 62.04 73.671 62.254 73.654 62.549 C 73.672 62.894 73.865 63.138 74.227 63.282 C 74.589 63.426 75.008 63.563 75.485 63.695 C 75.961 63.827 76.38 64.051 76.742 64.362 C 77.105 64.673 77.292 65.167 77.312 65.847 C 77.286 66.731 76.982 67.39 76.396 67.82 C 75.809 68.25 75.094 68.462 74.249 68.46 C 73.738 68.462 73.255 68.386 72.809 68.225 C 72.363 68.063 71.954 67.81 71.577 67.465 L 72.823 66.096 C 73.006 66.333 73.234 66.516 73.502 66.646 C 73.771 66.777 74.05 66.845 74.347 66.847 C 74.648 66.848 74.922 66.779 75.169 66.643 C 75.415 66.505 75.545 66.296 75.56 66.016 C 75.54 65.665 75.347 65.409 74.978 65.251 C 74.609 65.093 74.183 64.945 73.698 64.802 C 73.212 64.658 72.786 64.429 72.417 64.122 C 72.049 63.815 71.855 63.338 71.835 62.69 C 71.865 61.842 72.173 61.201 72.764 60.766 C 73.356 60.331 74.058 60.114 74.867 60.113 C 75.319 60.11 75.753 60.173 76.169 60.304 Z M 83.072 60.304 C 83.488 60.434 83.866 60.644 84.205 60.939 L 83.001 62.255 C 82.852 62.076 82.66 61.941 82.423 61.855 C 82.186 61.768 81.956 61.725 81.734 61.726 C 81.469 61.724 81.212 61.785 80.96 61.913 C 80.709 62.04 80.575 62.254 80.556 62.549 C 80.576 62.894 80.767 63.138 81.129 63.282 C 81.492 63.426 81.911 63.563 82.387 63.695 C 82.864 63.827 83.284 64.051 83.646 64.362 C 84.007 64.673 84.195 65.167 84.214 65.847 C 84.189 66.731 83.885 67.39 83.298 67.82 C 82.712 68.25 81.996 68.462 81.152 68.46 C 80.641 68.462 80.159 68.386 79.711 68.225 C 79.265 68.063 78.857 67.81 78.481 67.465 L 79.725 66.096 C 79.909 66.333 80.137 66.516 80.405 66.646 C 80.673 66.777 80.953 66.845 81.249 66.847 C 81.55 66.848 81.826 66.779 82.072 66.643 C 82.318 66.505 82.449 66.296 82.463 66.016 C 82.444 65.665 82.249 65.409 81.881 65.251 C 81.512 65.093 81.086 64.945 80.601 64.802 C 80.116 64.658 79.69 64.429 79.321 64.122 C 78.951 63.815 78.758 63.338 78.739 62.69 C 78.767 61.842 79.076 61.201 79.667 60.766 C 80.259 60.331 80.96 60.114 81.77 60.113 C 82.222 60.11 82.657 60.173 83.072 60.304 Z M 34.361 60.312 C 35.208 60.296 35.919 60.46 36.49 60.81 C 37.062 61.161 37.357 61.801 37.379 62.726 C 37.36 63.642 37.083 64.281 36.539 64.642 C 35.994 65.003 35.295 65.177 34.441 65.162 L 33.16 65.162 L 33.16 68.26 L 31.409 68.26 L 31.409 60.312 L 34.361 60.312 Z M 42.024 60.312 C 42.861 60.298 43.562 60.467 44.131 60.824 C 44.699 61.181 44.996 61.815 45.02 62.726 C 45.018 63.291 44.874 63.768 44.584 64.157 C 44.293 64.546 43.866 64.795 43.304 64.904 L 45.321 68.26 L 43.224 68.26 L 41.562 65.082 L 40.699 65.082 L 40.699 68.26 L 38.947 68.26 L 38.947 60.312 L 42.024 60.312 Z M 70.098 60.312 L 70.098 61.931 L 66.452 61.931 L 66.452 63.411 L 69.898 63.411 L 69.898 65.028 L 66.452 65.028 L 66.452 66.643 L 70.302 66.643 L 70.302 68.26 L 64.701 68.26 L 64.701 60.312 L 70.098 60.312 Z M 48.749 62.45 C 48.309 62.919 48.082 63.531 48.073 64.286 C 48.082 65.041 48.309 65.653 48.749 66.122 C 49.19 66.591 49.788 66.833 50.545 66.847 C 51.301 66.833 51.9 66.591 52.34 66.122 C 52.78 65.653 53.006 65.041 53.016 64.286 C 53.006 63.531 52.78 62.919 52.34 62.45 C 51.9 61.982 51.301 61.74 50.545 61.726 C 49.788 61.74 49.19 61.982 48.749 62.45 Z M 33.16 63.682 L 34.325 63.682 C 34.647 63.688 34.93 63.621 35.174 63.481 C 35.418 63.343 35.548 63.09 35.561 62.726 C 35.54 62.329 35.375 62.071 35.063 61.953 C 34.751 61.835 34.416 61.78 34.059 61.793 L 33.16 61.793 L 33.16 63.682 Z M 40.699 63.601 L 41.731 63.601 C 42.071 63.619 42.4 63.578 42.708 63.481 C 43.017 63.386 43.181 63.128 43.201 62.713 C 43.185 62.326 43.038 62.076 42.762 61.957 C 42.486 61.838 42.178 61.781 41.842 61.793 L 40.699 61.793 L 40.699 63.601 Z" transform="matrix(1, 0, 0, 1, 0, -8.881784197001252e-16)"/>
```